### ISSUE TITLE
Error running ResNet50 - Release 0.2 - incorrect argument label in call (have 'logits:labels:', expected 'logits:oneHotLabels:')

### ISSUE THREAD

**Issue by johndpope** [tensorflow/swift-models#67]
![Screen Shot 2019-03-20 at 21 39 16](https://user-images.githubusercontent.com/289994/54678246-c5d84100-4b58-11e9-9ddb-f71d68118962.png)

simple fix will submit PR.
```swift
return softmaxCrossEntropy(logits: logits, labels: oneHotLabels) -> 

return softmaxCrossEntropy(logits: logits, oneHotLabels: oneHotLabels)
```

**Comment by saeta** [comment]
Thanks for raising this issue, looking forward to your PR!

**Comment by johndpope** [comment]
I applied the update, created an Xcode project but it actually just gets stuck building.  I’m on a 13 in MacBook Pro / mojave 10.14.4 beta / Xcode 10.1 /tensorflow release 0.2 / this is stuck in limbo.
